image_tools/src/cam2image_main.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
We can remove this file now that rclcpp_component is handling building the executable.